view/frontend/templates/boltaccount.phtml

```diff
@@ -0,0 +1,96 @@
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/**
+ * Bolt Account button
+ *
+ * @var $block \Bolt\Boltpay\Block\Js
+ */
+if ($block->shouldDisableBoltCheckout()) return;
+if (!$block->isOrderManagementEnabled()) return;
+if ($block->isBlockAlreadyShown('account')) return;
+$accountJsUrl = $block->getAccountJsUrl();
+$checkoutKey = $block->getCheckoutKey();
+?>
+<div class="bolt-account-login"></div>
+<script>
+    require([
+        'jquery',
+        'matchMedia',
+    ], function ($, mediaCheck) {
+        var insertAccountScript = function () {
+            var scriptTag = document.getElementById('bolt-account');
+            if (scriptTag) {
+                return;
+            }
+            scriptTag = document.createElement('script');
+            scriptTag.setAttribute('type', 'text/javascript');
+            scriptTag.setAttribute('async', '');
+            scriptTag.setAttribute('src', '<?php echo $accountJsUrl; ?>');
+            scriptTag.setAttribute('id', 'bolt-account');
+            scriptTag.setAttribute('data-publishable-key', '<?php echo $checkoutKey; ?>');
+            document.head.appendChild(scriptTag);
+        }
+        mediaCheck({
+            media: '(max-width: 768px)',
+            entry: function () {
+                // When magento constructs mobile menu it copies elements instead of moving
+                // We need to wait for this coping, remove first bolt-account-login div
+                // And only ofter that insert account.js script
+                let timerId = setTimeout(function boltAccountLookup() {
+                    var $account_div = $("div.bolt-account-login");
+                    if ($account_div.length==2) {
+                        $account_div.eq(0).remove();
+                        insertAccountScript();
+                    } else {
+                        timerId = setTimeout(boltAccountLookup, 1000)
+                    }
+                }, 1000);
+            },
+            exit: function () {
+                // For desktop we are ready to insert script
+                insertAccountScript();
+            }
+        });
+
+    });
+</script>
+
+<style>
+    .bolt-account-login {
+        display: inline-block;
+        margin-left: 15px;
+    }
+
+    .header.panel > .header.links {
+        height: 50px;
+        line-height: 50px;
+    }
+
+    .header.panel > .header.links > li.welcome, .header.panel > .header.links > li > a {
+        line-height: 50px !important;
+    }
+
+    .header.panel > .header.links > li.customer-welcome > span > button {
+        margin-top: 14px;
+    }
+
+    .header.panel {
+        padding-top: 0px !important;
+        padding-bottom: 0px !important;
+    }
+</style>
```

Comment:
We should probably have no hardcoded CSS. This can all go into GlobalCSS config field as addition to the default value.

Reply:
1. We shouldn't apply these rules if feature is enabled.
2. If we add it into default value it doesn't affect to existing merchant

For (1) we can add an additional CSS class to .header.panel and add this class to rules. But what about (2).
What's your opinion @danac-gs

Reply:
@vitaliyreznikov correct, the (default) configuration fields won't update on plugin update. Perhaps we should ask merchants to update configuration manually in such cases. @matt-thomason @njfaries

Reply:
I propose to copy settings to GlobalCSS defaults but in this place check if GlobalCSS has any rules related to the feature and include CSS here if not. How it sounds @danac-gs?

Reply:
@vitaliyreznikov Personally, I don't have nothing against it, but recently we had a request to remove all hardcoded CSS from the plugin code. Moreover, assuming _.header.panel_ dimensions are the same in every theme / device can be wrong.

Reply:
@danac-gs is correct here, we should not have any hardcoded CSS in the plugin code, especially relating to bolt-created classes or items.
@vitaliyreznikov perhaps we can take a simpler approach, and include these items to copy into GlobalCSS as part of our onboarding documents, both public and private facing?

Reply:
Done. Created task about documentation - https://boltpay.atlassian.net/browse/M2P-34